frontend/scripts/react-components/dashboard-element/dashboard-widget/table-modal/table-modal.scss

```diff
@@ -0,0 +1,18 @@
+@import 'styles/settings';
+
+.c-table-modal {
+  width: 820px;
+  padding: 30px 0;
+  display: flex;
+  flex-direction: column;
```

Comment:
Do we need this? Most of the times `display: block;` and `margin: 0 auto` would do the trick. I'm saying this, because recently I realised I abuse `flex-direction: column`.

https://css-tricks.com/centering-css-complete-guide/

Reply:
I think flex is a better way of positioning. Also in fluid structures. Not sure how this is going to change

Reply:
Why is it better in this case? I think `flex-direction: column` should be only used if needed: it adds more to it's children than just centering.